### ISSUE TITLE
GCC7.1 usrsctp compilation errors

### ISSUE THREAD

**Issue by murillo128** [rawrtc/rawrtc-data-channel#2]
I am hitting this issue:

https://github.com/sctplab/usrsctp/issues/149

Could it be possible to update the dependency to include the fix?

**Comment by lgrahl** [comment]
Was tracked in https://github.com/rawrtc/rawrtc/issues/68 and @dbrgn posted a workaround (https://github.com/rawrtc/rawrtc/issues/68#issuecomment-326774904).

Blocked by https://github.com/NEAT-project/usrsctp-neat/issues/10

**Comment by murillo128** [comment]
Patched the .sh as suggested on the other issue:

```patch
diff --git a/make-dependencies.sh b/make-dependencies.sh
index fe842eb..cd1735f 100755
--- a/make-dependencies.sh
+++ b/make-dependencies.sh
@@ -75,6 +75,8 @@ if [ -z "$SKIP_USRSCTP" ]; then
     fi
     git checkout ${USRSCTP_BRANCH}
     git reset --hard ${USRSCTP_COMMIT}
+    echo ${USRSCTP_PATH}
+    sed -i 's/SCTP_DIAG_INFO_LEN 64/SCTP_DIAG_INFO_LEN 512/'  usrsctplib/netinet/sctp_constants.h
     cd ${MAIN_DIR}
 fi

```

Due to submodules  128 was even small for me :)

**Comment by lgrahl** [comment]
We should probably add this path to this project until the issue has been resolved.

**Comment by lgrahl** [comment]
Resolved by #28
